stores/migrations_utils.go

```diff
@@ -0,0 +1,79 @@
+package stores
+
+import (
+	"fmt"
+
+	gormigrate "github.com/go-gormigrate/gormigrate/v2"
+	"go.sia.tech/renterd/api"
+	"go.uber.org/zap"
+	"gorm.io/gorm"
+)
+
+// initSchema is executed only on a clean database. Otherwise the individual
+// migrations are executed.
+func initSchema(db *gorm.DB, metrics bool, logger *zap.SugaredLogger) gormigrate.InitSchemaFunc {
```

Comment:
Imo a `bool` is not a good fit here. I'd just pass in either `main` or `metrics` as strings and then log `logger.Infof("initialising '%s' schema", dbName)`